cpp/teensy/lib/multilateration/multilateration.cpp

```diff
@@ -0,0 +1,125 @@
+#include "multilateration.h"
+
+/*
+TODO:
+- clean up main (delete all)
+
+*/
+
+arm_status calculatePingerPosition(int32_t TdoaArray[],
+                                   HydrophonePositions HydrophonePositions[],
+                                   const arm_matrix_instance_f32* pA,
+                                   const arm_matrix_instance_f32* pB,
+                                   arm_matrix_instance_f32* pResult) {
```

Comment:
IMO the result of this function should be a position. 

```suggestion
arm_status calculatePingerPosition(int32_t TdoaArray[],
                                   HydrophonePositions HydrophonePositions[],
                                   const arm_matrix_instance_f32* pA,
                                   const arm_matrix_instance_f32* pB,
                                   Position* pResult) {
```